README.md

```diff
@@ -11,13 +11,18 @@ An attempt at collecting some of the collections of accessible components but al
 - [A11y Project Patterns](http://a11yproject.com/patterns.html) - An accessible widget & pattern library
 - [Reakit](https://reakit.io/) - Accessible, Composable and Customizable components for React
 
+## Contrast checker
+
+- [a11y-contrast](https://github.com/darekkay/a11y-contrast) — A CLI utility to calculate/verify accessible magic numbers for a color palette.
```

Comment:
This is a tool for measuring contrast and not a component so I don't think it fits the list. Sorry!

Reply:
Makes sense. Checking from my smartphone--I'll try to update next time I get to a proper laptop